### ISSUE TITLE
Script should not hang when combo-box not found

### ISSUE THREAD

**Issue by sguptatw** [getgauge/taiko#54]
**Expected behavior**
Should be able to locate the given element

**Steps to replicate**
* Add a file issue.js
```
const { browser, openBrowser, goto, click,comboBox } = require('taiko');
const assert = require("assert");

(async () => {
    try {
        await openBrowser();
        await goto("google.com");
        await click("I'm Feeling Lucky",{waitForNavigation:true});
        await click(link("About"));
        assert.ok(await comboBox(toRightOf(image("id","lang-icon"))).exists())
        await comboBox(toRightOf(image("id","lang-icon"))).select("hi");
        await click("Doodles Archive",{waitForNavigation:true});
        await closeBrowser();
    } catch (e) {
        console.error(e);
    } finally {
    }
})();
```
* Run script `taiko issue.js`
> Combo box is not found and the script hangs
**Version**
```
Version: 0.0.9 ()
```
Blocks #29

**Comment by Apoorva-GA** [comment]
Fixed as part of 24bb785fdf89409ee01271d0385c0a25c6525f55

**Comment by sguptatw** [comment]
Getting combo-box not found and the script hangs
```
Error: Combo Box not found
    at Object.select (...\lib\taiko.js:749:32)
```
